src/libraries/Common/src/System/Security/Cryptography/IncrementalHash.netfx.cs

```diff
@@ -0,0 +1,227 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
```

Comment:
(Outloud thinking)

I wonder if it would make sense to have an

```C#
#if !NETFRAMEWORK || NET471_OR_GREATER
#error "This implementation of IncrementalHash should only be used for .NET Framework where it is not available"
#endif
```

So that it doesn't accidentally get used someplace it shouldn't.